doc/coding-style-guide.md

```diff
@@ -220,4 +221,12 @@ So here are some basic guidelines:
 
 [Common guidelines](https://github.com/golang/go/wiki/CodeReview)
 
-// TODO
+- (🧑‍🤝‍🧑) 2 approvals per PR is preferred
+- (✅) Resolving a comment is the duty of the commenter. (after the comment was addressed)
+- (😬) When nitpicking/complaining always provide possible solutions, otherwise avoid commenting about it.
+- (🧑‍💻) Run the PR locally/on-your-environment if possible.
+   - (🚨) If testing steps not-clear/not-provided notify the creator to improve them
+   - (🚦) After running it "locally" notify the creator by commenting `Ran it, LGTM` (or something similar) or `Found possible issue ...`
```

Comment:
you can add well-known comment `practically LGTM`.

Reply:
done